lib/krikri/random_record_generator.rb

```diff
@@ -1,19 +1,36 @@
 module Krikri
   # Gets random records from the search index
   class RandomRecordGenerator
+    include Krikri::QaProviderFilter
 
     def initialize
-      @solr_repo = Blacklight::SolrRepository.new(Blacklight::Configuration.new)
+      @index_querier = Krikri::IndexQuerier.new
+      @default_query_params = { :id => '*:*',
```

Comment:
Does this make Solr search and return all of a provider's documents, and then randomly choose an offset from that list? If so, is there any way to point Solr directly at a random offset without retrieving a potentially large number of documents?

I think we do that with ElasticSearch somewhere, and I know it is done with CouchDB in the old system (not that CouchDB has anything to do with this, but it's a feature), and relational DBs.  I could also be reading this wrong.